PythonWrapperGenerator/src/APIData.h

```diff
@@ -70,9 +70,14 @@ struct ClassData
 		QList<ClassAttribute> attributes_;
 		QList<ClassMethod> methods_;
 		QList<EnumData> enums_;
+		QString usingAlias_{};
```

Comment:
Just FYI, whether you have a `{}` here or not makes no difference. I generally prefer not having it (less code).